### ISSUE TITLE
Add keyboard support for slider

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco-CMS#3957]
I think it would be great to add keyboard suppport when the handle has focus as in the example.
https://refreshless.com/nouislider/examples/#section-keyboard

There is also an discussion here about keyboard support with two handles:
https://github.com/leongersen/noUiSlider/issues/203

I also found this gist to improve the accessibility of noUiSlider.
https://gist.github.com/felixgirault/3fcb18ee0268b1c3127c

**Comment by bjarnef** [comment]
@nul800sebastiaan any chance we can update `nouislider` to version 13.0.0, which now comes with keyboard support build in,
https://github.com/umbraco/Umbraco-CMS/blob/1090d54763deb6ca69ca609476fa19b041fca388/src/Umbraco.Web.UI.Client/package.json#L38

https://github.com/leongersen/noUiSlider/issues/724#issuecomment-461009737

**Comment by nul800sebastiaan** [comment]
@bjarnef Sure, go for it!

**Comment by bjarnef** [comment]
@nul800sebastiaan of course that would be version 13.1.5 at the time of writing.
https://github.com/leongersen/noUiSlider/releases

**Comment by nul800sebastiaan** [comment]
Yeah, all good, v13 does not have any breaking changes listed so should be fine.

**Comment by bjarnef** [comment]
I will have a look at it.

**Comment by bjarnef** [comment]
PR submitted: https://github.com/umbraco/Umbraco-CMS/pull/5651
